### PR TITLE
Feature/54395 test get leads

### PR DESCRIPTION
**Pull request by christiandp7**
Adding Leads endpoint test [GET]

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+
+app.get('/version', async (_req: Request, res: Response): Promise<Response> => {
+  return res.status(200).send({
+    version: process.env.npm_package_version
+  });
 });
 
-server.listen(port, () => {
-  console.log(`Server running at port ${port}`);
+app.get('/leads', async (_req: Request, res: Response): Promise<Response> => {
```
Comment (marcelo-yourday) [inline src/index.ts:19]: @christiandp7 if we add all REST API resources and verbs we will have a highly coupled file. Maybe it is worth to try to decouple the service layer. Maybe you can take a look to this:

https://expressjs.com/en/4x/api.html#router

**Review by marcelo-yourday — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (christiandp7) [reply src/index.ts:19]: Thanks @marcelo-yourday The team is already working on this refactor, I'm currently working on Unit Testing files. I'll fix the branch conflict locally and then push the changes.

**Review by marcelo-yourday — APPROVED** [review]

**Comment by christiandp7** [comment]
@marcelo-yourday @douglasalizares Please review my latest changes, I've Fixed server.close() and fix some git conflicts.

**Review by douglasalizares — APPROVED** [review]
looks good.

**Pull request merged**